h/templates/deform/sequence_item.jinja2

```diff
@@ -1,15 +1,25 @@
 {% if not field.widget.hidden %}
 <li class="list-input__item" title="{{ field.description }}">
 
-  {{ field.serialize(cstruct=cstruct) | safe }}
+  <div class="list-input__item-inner">
```

Comment:
`list-input__item-inner` is the horizontal row containing the field and the remove item button.